Assets/Scripts/Cell.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+
+namespace Ksen
+{
+    public abstract class Cell : MonoBehaviour
+    {
+
+
+        public abstract void SetTextValue(string text);
```

Comment:
really good construction of an abstract class + child classes, we will need to discuss the implementation later.